examples/gasprice-retry/app.ts

```diff
@@ -0,0 +1,37 @@
+// tslint:disable: no-console
+
+import * as cutil from 'util'
+
+import { TransactionClient } from '../../src'
+
+export const start = async () => {
+  try {
+    const txClient = new TransactionClient('http://localhost:8031')
+
+    // Try to send a transfer transaction with a lower gas price than the network requires
+    // Increment a retry every 10s by sending a 10% increase of the gasPrice each time until reaching 40% increase
+    const res = await txClient.transfer(
+      {
+        chain: 'besu',
+        params: {
+          from: '0x7e654d251da770a068413677967f6d3ea2fea9e4', // Default Orchestrate account in development mode
+          to: '0x6009608a02a7a15fd6689d6dad560c44e9ab61ff',
+          value: '100000000',
+          gasPrice: '1000000000',
+          gasPricePolicy: {
+            retryPolicy: {
```

Comment:
IMO it would be more valuable for understanding to use `priority` instead of `gasPrice` here